docs/detections/session-view.asciidoc

```diff
@@ -16,20 +16,23 @@ and investigating session activity on your Linux infrastructure and understandin
 * *Process and event telemetry:* Process information included in the Linux logical event model.
 * *Nested sessions:* Sessions started by processes descended from the entry session.
 * *Alerts:* Alerts in the context of the processes which caused them.
+* *TTY output:* Terminal output associated with each process in the session.
 
 NOTE: To view Linux session data from your Kubernetes infrastructure, you'll need to set up the <<kubernetes-dashboard,Kubernetes dashboard>>.
 
 [float]
 [[enable-session-view]]
 === Enable Session View data
 Session View uses process data collected by the {elastic-defend} integration,
-but this data is not collected by default. To enable Session View data, go to *Manage* -> *Policies*
+but this data is not collected by default. To enable Session View data: go to *Manage* -> *Policies*
```

Comment:
Starting from 8.5 Session data is enabled by default when the user selects Cloud Workload integrations.

![image](https://user-images.githubusercontent.com/19270322/195433855-86a3b348-fc85-442e-82bf-938ee1f72b21.png)


But the steps are still valuable for enabling it in Endpoint integrations and to enable TTY output